page/using-jquery-core/avoid-conflicts-other-libraries.md

```diff
@@ -123,7 +123,7 @@ jQuery.noConflict();
 </script>
 ```
 
-Note that if you use this technique, you will not be able to use prototype.js methods inside the immediately invoked function that expect `$` to be prototype.js's `$`.
+Note that if you use this technique, you will not be able to use prototype.js methods inside the immediately invoked function. `$` will be a reference to jQuery, not prototype.js.
```

Comment:
If you are referring to 'prototype.js', I would rather leave it that way so as to avoid confusing the library and with the concept of prototype.